buildbot/configure.py

```diff
@@ -251,16 +251,9 @@ def do_configure(args, passthrough_args):
     cmake_cmd.append(llvm_dir)
 
     if args.use_libcxx:
```

Comment:
Please, remove `--libcxx-include` and `--libcxx-library` options. This script is not part of the deployed product, so depreciation policy we apply to the compiler/runtime features is not applicable.

Alternative option is to add a warning for using `--libcxx-include` and/or `--libcxx-library`. New version of the script ignores these options and uses libc++ from the source tree. IMO, it's not worth bothering.